src/js/port_handler.js

```diff
@@ -233,8 +246,6 @@ PortHandler.detectPort = function() {
             TABS.firmware_flasher.boardNeedsVerification = true;
         }
 
-        self.portPickerElement.trigger('change');
```

Comment:
👍 these are the main things we need to remove. We have a lot of code which mixes logic with UI heavily.

Reply:
Yes, that is what I tried. Specially if they affect Vue components.